pandas/core/frame.py

```diff
@@ -9805,7 +9805,8 @@ def join(
 
         Parameters
         ----------
-        other : DataFrame, Series, or list of DataFrame
+        other : DataFrame, Series, or a list containing any combination of DataFrames
+            and Series.
```

Comment:
code checks build is failing with 

> Error: /home/runner/work/pandas/pandas/pandas/core/frame.py:9789:PR08:pandas.DataFrame.join:Parameter "other" description should start with a capital letter

I think it's confused by the period here; hopefully removing it should resolve.